Filipo0on/chessgame
# Feature/apollo client

This PR is to enable client to connect to server.

**To test this PR:**
_Open two new terminal windows pointing at_ **server folder**
3. Run `json-server -p 1337 --watch db.json` in on of them.
4. Run `npm run dev` in the other

Your server should now be up and running!

_Open one terminal window pointing at_ **client folder**
1. First you need to run `npm install` in the client folder, to get the new dependencies installed.
2. Run `npm start`

Your app should be up and running, at http://localhost:3000/
Open the console, you should now see a log with an object like this:
`data {getGames: Array(3)}`

This contains the data from GET_GAMES query.

_This query will probably not stay in the lobby file, it is just a basic test, and to show the basic query request structure._
